generalInfoPage/general.html

```diff
@@ -25,6 +25,16 @@
           </div>
 
           <div>
+
+            <ul id="list">
+                <li><a href="../index.html">home</a></li>
+                <li id="search"><a href="search.html">search</a></li>
+                <li><a href="../aboutPage/about.html">about us</a></li>
+            </ul>
+
+        </div>
+
+          <div id="menu">
               <button onclick="show()">
                 <i id="menu-icon" class="fas fa-align-right fa-flip-vertical"></i>
```

Comment:
the menu icon didn't have the same style on all pages
in some, it misses font-size: 1.5rem;